packages/office-ui-fabric-react/src/components/TextField/TextField.test.tsx

```diff
@@ -331,19 +331,19 @@ describe('TextField with error message', () => {
     return delay(20).then(() => assertErrorMessage(wrapper!.getDOMNode(), errorMessage));
   });
 
-  it('should render error message when onGetErrorMessage returns a Promise<JSX.Element>', () => {
-    function validator(value: string): Promise<string | JSX.Element> {
-      return Promise.resolve(value.length > 3 ? errorMessageJSX : '');
-    }
+  // it('should render error message when onGetErrorMessage returns a Promise<JSX.Element>', () => {
```

Comment:
You might consider disabling this test by marking it as `xit` vs. `it` rather than commented out. This way it is reported as skipped.

Reply:
Oh cool, I didn't know about that!